tests/unit/IonTextWriterTest.js

```diff
@@ -358,6 +358,43 @@
           writer => { writer.writeStruct(); writer.endContainer(); writer.writeStruct() },
           '{}\n{}');
 
+      //Annotations
+
+      writerTest('Escapes keyword null in an annotation',
```

Comment:
Tests look good, 

Add another one for text with spaces, e.g. `"A text with spaces"`